packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -894,6 +894,14 @@ const ComboBox = forwardRef(
       if (!inputRef.current?.value && evt.type === 'blur') {
         selectItem(null);
       }
+
+      // Close the dropdown when focusing on the AI label or Toggletip
+      if (evt.type === 'focus' && (
+        evt.target.closest(`.${prefix}--ai-label`) ||
+        evt.target.closest(`.${prefix}--toggletip`)
```

Comment:
Are there alternative methods to determine what's focused besides checking for class selectors? That approach seems fragile.